rules/no-for-loop.js

```diff
@@ -5,6 +5,7 @@ const isLiteralValue = require('./utils/is-literal-value');
 const avoidCapture = require('./utils/avoid-capture');
 const getChildScopesRecursive = require('./utils/get-child-scopes-recursive');
 const singular = require('./utils/singular');
+const {getStaticValue} = require('eslint-utils');
```

Comment:
See the first require statement.

Reply:
Fixed. I'm used to the [import/no-duplicates](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md) rule catching/autofixing that for me in some of my own codebases.